Plash/AppDelegate.swift

```diff
@@ -443,4 +445,30 @@ final class AppDelegate: NSObject, NSApplicationDelegate {
 
 		menu.addQuitItem()
 	}
+
+	/**
+	Scans the given URL for placeholders and subtitutes them with
+	their corresponding values.
+	*/
+	private func replacePlaceholders(of url: URL) -> URL? {
+		// Here we swap out [[screenWidth]] and [[screenHeight]] for their actual values.
+		// We proceed only if we have an NSScreen to work with.
+		guard let screen = desktopWindow.targetScreen?.withFallbackToMain ?? .main else {
+			print("No screen was found to read dimensions from!")
+			return nil
+		}
+		let screenWidthPlaceholder = URLPlaceholder(token: "[[screenWidth]]") { () -> String in
+			String(format: "%.0f", screen.visibleFrameWithoutStatusBar.width)
+		}
+		let screenHeightPlaceholder = URLPlaceholder(token: "[[screenHeight]]") { () -> String in
+			String(format: "%.0f", screen.visibleFrameWithoutStatusBar.height)
+		}
+
+		do {
+			return try url.replacingPlaceholder(screenWidthPlaceholder).replacingPlaceholder(screenHeightPlaceholder)
+		} catch {
+			print(error.localizedDescription)
```

Comment:
Use `error.presentAsModal()`.

Reply:
I don't see a method with that name. Where can I find that?

Reply:
I found the definition in Gifski and brought it into `Utilities.swift`.

Reply:
Oh, sorry. I thought I had it here. Good find though 👍🏻